README.md

```diff
@@ -3,7 +3,7 @@
 ## ~~0. Install via Maven or Gradle:~~
 > Note: The PlayFab SDK Maven package is out of date. We are working to update to the latest. For now, to get the latest Java SDK for your game, you will need to clone this repo and build from source.
 
-Check the [Maven Central Repository](https://search.maven.org/#search%7Cga%7C1%7Ccom.playfab) for latest versions of our SDKs.  
+Check the [Public Artifact Repository](https://dev.azure.com/PlayFabInternal/Main/_artifacts/feed/PlayFab-JavaSDK-combined) for latest versions of our SDKs.  
```

Comment:
Can you confirm this url is publicly accessible? Should we be hosting the artifact on GitHub instead?

Reply:
I can confirm a private browser window without logging into ado can access the jar files from the url.

GitHub sounds like it would be more official and may not require the README update as is if we are able to automate packaging through ADO.

Reply:
Alright. I'm fine hosting this on ADO.